client/src/components/FAQs/FAQs.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react'
+import './FAQs.css'
+
+class FAQs extends Component {
+ 
+  render () {
```

Comment:
@jatin11gidwani  Components are just presentation.They will simply return the formatted data(which will be passed as props)  which will be rendered from the container.No need for `render()` here just `return` it.
`return (<div>FAQs</div>)`
 Also we need to define PropTypes
Please refer [SampleComponent.js](https://github.com/UdacityMobileWebScholarship/guess-quote/blob/master/client/src/components/SampleComponent/SampleComponent.js)

Reply:
Also, do end the file with a new line "Always" :)